Makefile

```diff
@@ -5,9 +5,6 @@ FLAGS=
 checkrst:
 	python setup.py check --restructuredtext
 
-pyroma:
-	pyroma -d .
-
 
 flake:checkrst pyroma
```

Comment:
This `pyroma` dependency target seems forgotten here.

Reply:
addressed in #737